viewing_party/party.py

```diff
@@ -1,23 +1,131 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie = {
+        "title": title, 
+        "genre": genre,
+        "rating": rating
+    }
+
+    if None in movie.values():
+        return None

```

Comment:
This is a really clever way to check the values without having to write the explicit checks. It's a little surprising from a code ordering perspective, though, as I generally expect to see error handling come first. This would definitely be a pattern you'd want to discuss as a team to see how everyone feels about it!

Be careful of the actual check on line 10. The function description says this should return None if any of the values are falsy. `None` is one falsy value, but there are others. We might try something like
```python
        if not all(movie.values()):
            return None
```